src/main/java/com/team4/testingsystem/controllers/AnswerController.java

```diff
@@ -0,0 +1,22 @@
+package com.team4.testingsystem.controllers;
+
+import org.springframework.web.bind.annotation.GetMapping;
+import org.springframework.web.bind.annotation.PostMapping;
+import org.springframework.web.bind.annotation.RequestBody;
+import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.bind.annotation.RequestParam;
+import org.springframework.web.bind.annotation.RestController;
+
+@RestController
+@RequestMapping("/answer")
+public class AnswerController {
+
+    @GetMapping("/essay")
+    public String downloadEssay(@RequestParam Long testId) {
```

Comment:
the frontend shouldn't have a separate endpoint for getting essay. The data should be taken on our side and passed to DTO